ql/src/Security/CWE-020/SuspiciousCharacterInRegexpGood.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import "regexp"
+
+func fixed(hostNames []byte) string {
+	var htmlRe = regexp.MustCompile("\\bforbidden.host.org")
```

Comment:
Perhaps we could recommend using raw string literals here? They're used in the `regexp` documentation and seem a bit nicer.

```suggestion
	var htmlRe = regexp.MustCompile(`\bforbidden.host.org`)
```